tests/test_arrow.py

```diff
@@ -57,9 +56,9 @@ def test_init(self):
         assert result._datetime == self.expected
 
     # regression tests for issue #626
-    def test_init_pytz_timezone(self):
```

Comment:
Can we add a separate test for pytz and another for zoneinfo? I don't think we should be removing pytz and closing the open issue for the sake of the tests. There is still a use-case for passing in a pytz timezone into Arrow, especially if someone has not yet adopted zoneinfo.

If pytz is not used in the core arrow code and only in the test code (it is only pulled in via `requirements-tests.txt`, I think we need another CR that focuses on adding support for Zoneinfo natively throughout rather than using dateutil, which is our primary means of doing timezones.

Reply:
Sure